dotnet-dapper/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentityWithDapper/Methods/Methods.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+
+namespace ReturnInsertedIdentityWithDapper;
+
+public static class Methods
+{
+    public static int UseOfOutputInserted(string connectionString, Student newStudent)
+    {
+        using var dbConnection = new SqlConnection(connectionString);
+
+        dbConnection.Open();
+
+        var lastInsertedId = dbConnection.QuerySingle<int>(
+            @$"INSERT INTO Students (Firstname, Surname) 
+               OUTPUT INSERTED.Id 
+               VALUES (@Firstname, @Surname);",
```

Comment:
No need for this to be an interpolated string nor a verbatim string. The suggestion was for this to be a parameterized query using raw string literals.